packages/epo-react-lib/src/atomic/Button/styles.ts

```diff
@@ -13,6 +13,7 @@ export const Button = styled.button<StyledButtonProps>`
   ${({ $styleAs = "primary" }) => aButtonTheme($styleAs)}
   ${aButton}
   align-items: center;
+  gap: 10px;
```

Comment:
use flex to set the gap between the icon + text instead of relying on sibling selectors